client/task-list/tasks/payments/PaymentGatewaySuggestions/components/List/test/list.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+//import { useSlot } from '@woocommerce/experimental';
```

Comment:
Don't forget to clean up this commented line :)

Reply:
Whoops!